packages/react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -47,6 +47,7 @@ export interface IContextualMenuProps
   /**
    * Optional callback to access the IContextualMenu interface. Use this instead of ref for accessing
    * the public methods and properties of the component.
+   * @deprecated ContextualMenu has no public methods, so componentRef no longer returns a ref
```

Comment:
This is, technically, not true, it just returns a `ref` to an empty object, doesn't it?

```suggestion
   * @deprecated ContextualMenu has no imperative methods in its public interface, so componentRef returns a useless ref to an empty object
```

Reply:
pretty sure it returns undefined right now

Reply:
```suggestion
   * @deprecated ContextualMenu has no imperative methods in its public interface, so componentRef no longer returns a ref
```